src/nanoemoji/write_font.py

```diff
@@ -23,7 +23,7 @@
 import dataclasses
 import math
 from fontTools import ttLib
-from fontTools.misc.arrayTools import rectArea, unionRect
+from fontTools.misc.arrayTools import rectArea, normRect, unionRect
```

Comment:
aside: We should consider implementing these on the picosvg Rect, would be quite handy there